python/composio/utils/enums.py

```diff
@@ -2,7 +2,7 @@
 
 
 def get_enum_key(name: str) -> str:
-    characters_to_replace = [" ", "-", "/", "(", ")", "\\", ":", '"', "'", "."]
+    characters_to_replace = [" ", "-", "/", "(", ")", "\\", ":", '"', "'", ".", "&"]
```

Comment:
Consider defining this list as a module-level constant to improve maintainability and reusability. Example:
```python
SPECIAL_CHARACTERS = [" ", "-", "/", "(", ")", "\\", ":", '"', "'", ".", "&"]

def get_enum_key(name: str) -> str:
    for char in SPECIAL_CHARACTERS:
        name = name.replace(char, "_")
    return name.upper()
```